### DISCUSSION TITLE
Any tentative date for next release to support elastic 8.4.0 & 8.4.1 serious

### DISCUSSION THREAD

**Discussion by AH-Siva** [fbaligand/kibana-enhanced-table#284] [General]
Hi @fbaligand - We been using the Enhanced table for last **3.5** years. This is really useful and effective for our monitoring setup. In our organization planned to upgrade ELK stack to **8.4.1**. When we checking the Enhanced table release, we could see its available till for 8.3 serious. Is there any ETA placed for 8.4 serious release? It would be more helpful if you could share the ETA for next release. 

Thank You,
Siva

**Comment by fbaligand** [comment]
Hi @AH-Siva,

Happy to see that you use enhanced-table for so long time.
Well, i currently work on next release that will include compatibility for Kibana 8.4.
I can't give you an ETA, but I think that release should happen during next week.

**Comment by fbaligand** [comment]
Hi @AH-Siva,

I have good news! Enhanced Table plugin v1.13.3 is out with support for Kibana 8.4:
https://github.com/fbaligand/kibana-enhanced-table/releases/tag/v1.13.3
